base_layer/wallet/src/error.rs

```diff
@@ -85,6 +86,16 @@ pub enum WalletError {
     UtxoScannerError(#[from] UtxoScannerError),
 }
 
+pub const ERROR_LOG_TARGET: &str = "tari::application";
```

Comment:
It's `LOG_TARGET` everywhere else in the code base

```suggestion
pub const LOG_TARGET: &str = "tari::application";
```